update/upgrade_db.sh

```diff
@@ -73,58 +120,101 @@ fi
 echo "Dumping actual DB"
 TODAY=`date '+%Y%m%d'`
 /usr/bin/pg_dump --host $HOST --port 5432 --username "$USER" --format custom --file "$TODAY""_current_qwat.backup" "$SRCDB"
-#/usr/bin/pg_dump -d "service=$QWATSERVICE" --format custom --file "$TODAY""_current_qwat.backup"
 
-echo "Droping existing qwat_test"
+echo "Dropping existing DB qwat_test"
 /usr/bin/dropdb "$TESTDB" --host $HOST --port 5432 --username "$USER" --no-password
-#/usr/bin/dropdb -d "service=$QWATSERVICETEST"
 
-echo "Creating DB (qwat_test)"
+echo "Creating DB qwat_test"
 /usr/bin/createdb "$TESTDB" --host $HOST --port 5432 --username "$USER" --no-password
-#/usr/bin/createdb -d "service=$QWATSERVICETEST"
 
-echo "Restoring in test DB"
+echo "Restoring current DB in qwat_test"
 /usr/bin/pg_restore --host $HOST --port 5432 --username "$USER" --dbname "$TESTDB" --no-password --single-transaction --exit-on-error "$TODAY""_current_qwat.backup"
```

Comment:
Create DB with template is not possible when there is a session opened on it. Would it be a problem ?